docs/ISSUE_TEMPLATE.md

```diff
@@ -19,7 +20,7 @@ Please provide an ordered list on how to reproduce the issue.
 
 **Steps to Reproduce**:
 
-if relevant, please include a screenshot.
+if relevant, please include a screenshot and logs.
```

Comment:
nit: **If** (with a capital I)

nit: please include ~a~ **screenshots** and logs. The point is - the "a" seems to be holding for both screenshot and logs, making it sound weird.

Reply:
Sure!! 😆